en/Data Structures/Tries/tries.md

```diff
@@ -0,0 +1,158 @@
+# Trie
```

Comment:
Except this. One of these files is redundant, could you please remove it?

Reply:
done